src/lando/conftest.py

```diff
@@ -464,3 +484,15 @@ def create_state_handler(revision, landing_assessment=None):
         )
 
     return create_state_handler
+
+
+@pytest.fixture
+def treestatus_url():
+    """A string holding the Tree Status base URL."""
+    return "http://treestatus.test"
```

Comment:
This should be using `settings.TREESTATUS_URL`. It can be overridden in `test_settings` if needed.

Reply:
This is pre-existing code that has been moved to the global conftest. I will update it.